packages/quantic/force-app/main/default/lwc/quanticDocumentSuggestion/quanticDocumentSuggestion.js

```diff
@@ -101,6 +122,15 @@ export default class QuanticDocumentSuggestion extends LightningElement {
     this.template.removeEventListener('rating', this.onRating);
   }
 
+  validateProps() {
+    if (!(Number(this.maxDocuments) > 0)) {
+      this.renderingError = `${this.maxDocuments} is an invalid maximum number of document suggestions. A integer greater than 0 was expected.`;
```

Comment:
I'd suggest to surround the interpolated value with double quotes as it makes it easier to spot leading or trailing white space issues in the error message.